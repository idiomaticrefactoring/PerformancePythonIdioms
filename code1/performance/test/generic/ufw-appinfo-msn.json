[{"profile":"MSN","title":"MSN Chat","description":"MSN chat protocol (with file transfer and voice)","tcp_list":[1863,6901],"udp_list":[1863,6901],"tcp_ranges":[{"start":6891,"end":6900}],"normalized_tcp_list":[1863,6901],"normalized_tcp_ranges":[{"start":6891,"end":6900}],"normalized_udp_list":[1863,6901]}]
