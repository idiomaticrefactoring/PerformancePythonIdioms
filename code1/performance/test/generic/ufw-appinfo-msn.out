Profile: MSN
Title: MSN Chat
Description: MSN chat protocol (with file transfer and voice)

Ports:
  1863
  6891:6900/tcp
  6901

